components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -237,4 +299,17 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     // Drop the previous fragment and add the incoming fragment.
     this.router.navigate([this.url.split('#')[0]], { fragment: event.target.value });
   }
+
+  onProjectChecked(project: ProjectChecked): void {
+    this.projects[project.id] = project;
```

Comment:
maybe a quick clarifying comment: `updates whether the project was checked or unchecked`